website/docs/from-typescript.md

```diff
@@ -0,0 +1,326 @@
+---
+id: from-typescript
+title: TypeScript ↔ Sorbet Reference
+sidebar_label: TypeScript ↔ Sorbet
+---
+
+Many people learning Sorbet for the first time come from a TypeScript
+background. We've built this quick reference table mapping some common
+TypeScript features to the corresponding feature in Sorbet. (This is not a
+document about automatically interoperating between Sorbet and TypeScript; it is
+merely provided for educational purposes.)
+
+Note that Sorbet is not affiliated with TypeScript in any official way, and the
+information in this table may become out of date as TypeScript evolves
+independently of Sorbet. Please use the "Edit" button at the top of the page to
+help correct inaccuracies.
+
+Also note that Sorbet features do not always map to an exactly equivalent
+TypeScript feature. Be sure to read the relevant Sorbet docs to learn more about
+the features Sorbet provides.
+
+<table>
+  <thead>
+    <tr>
+      <th>
+        TypeScript
+      </th>
+      <th>
+        Sorbet
+      </th>
+      <th>
+        Notes
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <code>string</code>
+      </td>
+      <td>
+        <code>String</code>
+      </td>
+      <td>
+        Ruby strings are represented by <a href="/docs/class-types">Class
+        Types</a>. All values in Ruby are instances of a particular class,
+        including strings.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <code>number</code>
+      </td>
+      <td>
+        <code>Integer</code>, <code>Float</code>, <code>Numeric</code>
+      </td>
+      <td>
+        Ruby has <a href="/docs/class-types">separate classes</a> for different
+        numeric types. <code>number</code> in JavaScript is most closely
+        <code>Float</code> in Ruby, but unlike in JavaScript, integers and
+        floating points numbers are not transparently compatible with each
```

Comment:
```suggestion
        floating-point numbers are not transparently compatible with each
```

"floating points numbers" sounds strange to me; I think the "s" should be dropped, but it's also possible that I have gotten the need for the hyphen incorrect.